contributors/devel/release.md

```diff
@@ -0,0 +1,231 @@
+lead](https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/features).
+
+Implementation and bugfixing is ongoing across the cycle, but
+culminates in a code slush and code freeze period:
+* The **code slush** starts in week ~9 of the release cycle.  The master
+  branch only accepts PRs for release milestone.  No additional feature
+  development is merged after this point.
+* The **code freeze** starts in week ~10 and continues for ~2 weeks.
+  Only critical bug fixes are accepted into the release codebase.
+
+After this point only critical bug fixes happen and all remaining
+issues are resolved before release.
+
+When the code base is sufficiently stable, the master branch re-opens for
+general development and the release is finalize from a branch.
+
+Following release, the Release Branch Manager cherry picks critical
+fixes from the master branch for a period of around 9 months, leaving
+an overlap of three release versions forward support.  Thus, each
+release is part of a broader kubernetes lifecycle:
+
+![Image of kubernetes release lifecycle spanning three releases](release-lifecycle.png)
+
+## Removal Of Items From The Milestone
+
+Before getting too far into the process for adding, please note:
+
+Members of the Release Team may remove Issues from the milestone
+if they or the responsible SIG determine that the issue is not
+actually blocking the release and is unlikely to be resolved in a
+timely fashion.
+
+Members of the Release Team may remove PRs from the milestone for
+any of the following, or similar, reasons:
+
+* PR is potentially de-stabilizing and is not needed to resolve a blocking issue;
+* PR is a new, late feature PR and has not gone through the features process or the exception process;
+* There is no responsible SIG willing to take ownership of the PR and resolve any follow-up issues with it;
+* PR is not correctly labelled;
+* Work has visibly halted on the PR and delivery dates are uncertain or late.
+
+While members of the Release Team will help with labelling and
+contacting SIG(s), it is the responsibility of the submitter to
+categorize PRs, and to secure support from the relevant SIG to
+guarantee that any breakage caused by the PR will be rapidly resolved.
+
+Where additional action is require, an attempt at human to human
+escalation will be made by the release team through the following
+channels:
+
+- Comment mentioning the sig team appropriate for the issue type
+- Emailing the SIG googlegroup list
+  - bootstrapped with the emails addresses from the [community sig list](/sig-list.md)
+  - maybe configured to email alternate googlegroup
+  - maybe configured to directly email SIG leads or other SIG members
+- Messaging the SIG slack channel, mentioning the SIG leads
+  - bootstrapped with the slackchannel and SIG leads from the [community sig list](/sig-list.md)
+  - maybe configured to message alternate slack channel and users
```

Comment:
ditto; I don't entirely understand this information. If a chain reaction of notifications is caused, I'm not even sure it belongs in this document - the most important point is that messages are sent and received. SIGs configuring their information channels is really up to them.

Reply:
Yeah this was all straight from the prior document, which was a mix of:
* letting the release team know what it should do relative to the dev community
* letting the dev membership understand what to expect from the release team

Reply:
I think it's reasonable, especially in the dev guide now to give a sense of, "Hey if you see the release team folks on about something..they're doing their escalation job."  Maybe something direct and informal like that conveys the sentiment better?